packages/neuron-wallet/src/controllers/api.ts

```diff
@@ -260,10 +260,20 @@ export default class ApiController {
       return SettingsService.getInstance().locale = locale
     })
 
-    handle('set-theme', async (_, theme: 'system' | 'light' | 'dark') => {
-      return SettingsService.getInstance().themeSource = theme
+    handle('is-dark', async () => {
+      return {
+        status: ResponseCode.Success,
+        result: nativeTheme.shouldUseDarkColors
+      }
     })
 
+    handle('set-theme', async (_, theme: 'system' | 'light' | 'dark') => {
+      SettingsService.getInstance().themeSource = theme
+      return {
+        status: ResponseCode.Success
+      }
+    })
```

Comment:
`enum ThemeEnum {
    SYSTEM = 'system',
    LIGHT = 'light',
    DARK = 'dark'
}`

Reply:
Are there any differences between these? Or why this is better?